iodata/inputs/orca.py

```diff
@@ -38,30 +39,31 @@
 *"""
 
 
+def default_atom_line(data, fields, iatom):
+    """Format atom line for ORCA input."""
+    symbol = num2sym[data.atnums[iatom]]
+    atcoord = data.atcoords[iatom] / angstrom
+    return f"{fields['indent']}{symbol:3s} {atcoord[0]:10.6f} {atcoord[1]:10.6f} {atcoord[2]:10.6f}"
```

Comment:
I can see how `fields` argument is used in the case of ORCA input files. Based on your experience with different programs, do you think `fields` would be commonly used in generating input files?

Alternatively, we can expect the user-provided `atom_line` function to take care of the indentation. I just find it simpler, if we don't need to provide `fields` to `atom_line` function...

Reply:
Other than the indentation, I cannot think of use cases. I don't believe it is critical in the sense that Orca will just run fine without proper indentation.

I'd suggest to remove the `fields` argument and the indentation. We can always reintroduce `fields` (as last argument) when we find a good reason for it. The only reason I put it there originally was to get exactly the same behavior before and after merging.

Reply:
This is fixed